std/uuid.d

```diff
@@ -732,15 +738,95 @@ public struct UUID
         /**
          * ditto
          */
-        @safe pure nothrow @nogc size_t toHash() const
+       @safe pure nothrow @nogc UUID opAssign(in UUID s)
         {
-            size_t seed = 0;
-            foreach(entry; this.data)
-               seed ^= cast(size_t)entry + 0x9e3779b9 + (seed << 6) + (seed >> 2);
+            ulongs[0] = s.ulongs[0];
+            ulongs[1] = s.ulongs[1];
+            return this;
+        }
 
-            return seed;
+        /**
+         * ditto
+         */
+        @safe pure nothrow @nogc UUID opAssign(ref in UUID s)
+        {
+            ulongs[0] = s.ulongs[0];
+            ulongs[1] = s.ulongs[1];
+            return this;
         }
 
+        /**
+         * ditto
+         */
+        //MurmurHash2
```

Comment:
1. From deprecated [site](https://sites.google.com/site/murmurhash/) _All code is released to the public domain. For business purposes, Murmurhash is under the MIT license._ From new [site](https://code.google.com/p/smhasher/) _All MurmurHash versions are public domain software, and the author disclaims all copyright to their code._
2. This PR and druntime are independent because UUID already has it's own `toHash` implementation. Another question is that finalisation _mix_ in AA implementation use only 32bit MurmurHash2. So I can try to implement 64bit MurmurHash2 finalisation too.